portfolio/src/main/webapp/script.js

```diff
@@ -190,8 +190,45 @@ function createNextButton(direction, isValid) {
   return nextButton;
 }
 
+/**
+ * Retrieves markers from /markers.
+ */
+function fetchMarkers() {
+  fetch('/markers').then(response => response.json()).then((markers) => {
+    markers.forEach(
+        (marker) => {
+            createGameMarker(marker.lat, marker.lng)});
+  });
```

Comment:
This can all go on one line `markers.forEach(marker => createGameMarker(marker.lat, marker.lng));